src/component/update/index.tsx

```diff
@@ -10,45 +10,33 @@ import "./index.css";
 
 interface UpdateProps extends DefaultUpdateProps {}
 
-type UpdateState = "text" | "graph";
-
 export function Component(props: UpdateProps) {
-  const [updateView, setUpdateView] = useState<UpdateState>("text");
-
-  useFlip("flip-root", {
-    duration: 1800
-  });
+  const [isFlipped, setIsFlipped] = useState(false);
 
   return (
-    <div data-flip-root-id="flip-root">
-      <AnimateInOut
-        out={{
-          from: { transform: "rotate(25deg)", opacity: 1 },
-          to: { transform: "rotate(65deg)", opacity: 1 },
-          duration: 1000
+    <ReactCardFlip isFlipped={isFlipped} flipDirection="vertical">
+      <PlasmicUpdate
+        {...props}
+        className={"update"}
+        state={"text"}
+        toggleUpdateView={{
+          "aria-label": "Toggle View",
+          onPress: () => {
+            setIsFlipped(!isFlipped);
+          }
         }}
-        in={{
-          from: { opacity: 1 },
-          to: { transform: "rotate(90deg)", opacity: 1 },
-          duration: 800
+      />
+      <PlasmicUpdate
+        {...props}
+        className={"update"}
+        state={"graph"}
+        toggleUpdateView={{
```

Comment:
We now have 2 components that each represent a front and back. Instead of changing state, we are assigning state to each element and turning them respectively 180 degrees on button press.